lib/app_profiler/railtie.rb

```diff
@@ -60,7 +60,7 @@ class Railtie < Rails::Railtie
         if (Rails.env.development? || Rails.env.test?) && AppProfiler.stackprof_viewer.remote?
           app.middleware.insert_before(0, AppProfiler.viewer::Middleware)
```

Comment:
i think we may need to ensure that this is inserted after rack events too, otherwise this middleware may be before the main one?